homeassistant/components/lcn/__init__.py

```diff
@@ -166,13 +180,13 @@ def get_connection(connections, connection_id=None):
 
     hass.data[DATA_LCN][CONF_CONNECTIONS] = connections
 
-    hass.async_create_task(
-        async_load_platform(hass, 'light', DOMAIN,
-                            config[DOMAIN][CONF_LIGHTS], config))
-
-    hass.async_create_task(
-        async_load_platform(hass, 'switch', DOMAIN,
-                            config[DOMAIN][CONF_SWITCHES], config))
+    # load platforms
+    for component, conf_key in (('cover', CONF_COVERS),
+                                ('light', CONF_LIGHTS),
+                                ('switch', CONF_SWITCHES)):
+        hass.async_create_task(
+            async_load_platform(hass, component, DOMAIN,
+                                config[DOMAIN][conf_key], config))
```

Comment:
I just realized that the platform config keys are optional, so we have to check for that before using it and loading the platform.

Reply:
Thanks! I should have seen this by myself.
Here we go: home-assistant/home-assistant#21354